cli/azd/internal/appdetect/java.go

```diff
@@ -165,48 +165,71 @@ func detectDependencies(currentRoot *mavenProject, mavenProject *mavenProject, p
 	for _, dep := range mavenProject.Dependencies {
 		if dep.GroupId == "com.mysql" && dep.ArtifactId == "mysql-connector-j" {
 			databaseDepMap[DbMySql] = struct{}{}
+			log.Printf("Detected 'db.mysql' because found this dependency in project: " +
+				"com.mysql:mysql-connector-j.")
 		}
 
 		if dep.GroupId == "org.postgresql" && dep.ArtifactId == "postgresql" {
 			databaseDepMap[DbPostgres] = struct{}{}
+			log.Printf("Detected 'db.postgres' because found this dependency in project: " +
```

Comment:
nit: we'd better maintain all these resource types in a map or something similar, so if we alter the name of the resource type later, we don't have to update these logs one by one.

Reply:
PR created: https://github.com/azure-javaee/azure-dev/pull/41

After refactor and refactor, I changed a lot in that PR.